student.rb

```diff
@@ -1,7 +1,7 @@
 require_relative './person'
 
 class Student < Person
-  def initialize(_classroom, age, name = 'Unknown', parent_permission: true)
+  def initialize(classroom, age, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission)
     @classroom = classroom
```

Comment:
When I create a student I get the following error.

![image](https://user-images.githubusercontent.com/38088818/193134904-e43a19f2-3366-4667-9427-12a6b22ab19c.png)

- This is due to the arguments passed in the `Student` class. Kindly look into it.